examples/box2d/bipedal_bdq.py

```diff
@@ -125,7 +128,9 @@ def save_best_fn(policy: BasePolicy) -> None:
         torch.save(policy.state_dict(), os.path.join(log_path, "policy.pth"))
 
     def stop_fn(mean_rewards: float) -> bool:
-        return mean_rewards >= getattr(env.spec.reward_threshold)
+        if env.spec and env.spec.reward_threshold:
```

Comment:
Yes true, it would have led to an error, but `stop_fn` is not being used in this example. For unknown reasons it's commented out when passed to the `OffpolicyTrainer` [here](https://github.com/thu-ml/tianshou/pull/1077/files/d01420c714d0a2bac82cd72027b8c7b3cca265ba#diff-3792a1a111e15c573f79326f5922fa97aad9f2022aa90d13a088ffada9b78434R153).

Reply:
Wanna fix this? :)

I guess someone accidentally broke it and then just commented it out

Reply:
The `epoch` was set by default to 1000. I guess (and hope) that the agent can reach `reward_threshold` way sooner (like 10-20 epochs as in other examples). I am running now the script to determine what the lowest epoch is s.t. `stop_fn` is true.